app/workers/scheduler/admin/dashboard_cache_warmer_scheduler.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+class Scheduler::Admin::DashboardCacheWarmerScheduler
+  include Sidekiq::Worker
+
+  # The values in this file have to align with the values in app/views/admin/dashboard/index.html.haml
+  # so that the generated cache keys are the same, otherwise cache warming will have no benefit
+
+  MEASURES = %w(
+    new_users
+    active_users
+    interactions
+    opened_reports
+    resolved_reports
+  ).freeze
+
+  DIMENSIONS = %w(
+    sources
+    languages
+    servers
+    software_versions
+    space_usage
+  ).freeze
+
+  DIMENSION_LIMIT = 8
+
+  TIME_PERIOD_DAYS = 29
+
+  RETENTION_TIME_PERIOD_MONTHS = 6
+
+  def perform
```

Comment:
Maybe add basic spec for this worker just showing that caches were not populated before but are populated after the `perform` execution?